File: syscall/dhcp.py
import os, sys

if __name__== '__main__':
	method = sys.argv[1]
	username = sys.argv[2]

if method == "run" :
	os.system("sh /var/www/shell/makeconfig.sh %s".format(username))
	os.system("sh /var/www/shell/makelease.sh %s".format(username))
	os.system("/var/www/server/dhcpark > /dev/null &");
	sys.exit(1)
elif method == "stop" :
	os.system("sh /var/www/shell/stopdhcpark.sh");
	sys.exit(1)
else:
	sys.exit(0)

File: api/dhcp.php
<?php 
/*
 *
 * LOGIN API CALL 
 * NEEDED PARAMETERS IN BODY : ID, PW 
 * RETURN 0 PASSWORD DOES NOT MATCH
 * RETURN SUBNET INFO OF CONFIG USER CREATED WHEN SUCCESSFUL QUERIED
 */
	header('Access-Control-Allow-Origin: http://192.168.0.2');

	$arr = array();

	$inputstring = $_SERVER['QUERY_STRING'];

	parse_str($inputstring, $input);
	$username = $input['username'];
	$method = strtolower( $input['method'] );
	if(!strcmp($method, "run")){
		exec("sudo -S sh /var/www/shell/makeconfig.sh $username");
		exec("sudo -S sh /var/www/shell/makelease.sh $username");
		exec("sudo -S /var/www/server/dhcpark >/dev/null &");
		echo json_encode('1');
		exit();
	}
	else if(!strcmp($method, "stop")){
		exec("sudo -S sh /var/www/shell/stopdhcpark.sh");
		echo json_encode('1');
		exit();
	}
	else if(!strcmp($method, "status")){
		exec("sudo -S sh /var/www/shell/dhcparkstatus.sh", $output, $error);
		echo json_encode($output);
		exit();
	}
	echo json_encode('0');
	exit();
?>
